src/api/types/generated-schema.ts

```diff
@@ -2389,8 +2389,17 @@ export interface components {
     SpeciesLookupDetailsResponsePayload: {
       /** @description List of known common names for the species, if any. */
       commonNames?: components["schemas"]["SpeciesLookupCommonNamePayload"][];
-      /** @description True if the species is known to be endangered, false if the species is known to not be endangered. This value will not be present if the server's taxonomic database doesn't indicate whether or not the species is endangered. */
-      endangered?: boolean;
+      /** @description IUCN Red List conservation category code. */
+      conservationCategory?:
```

Comment:
I cheated a bit on the schema change. There are also other schema changes related to observation results but they cause one of the observation functions to break, and I didn't want to clutter this PR with changes to that code. So I just cherry-picked the schema changes related to conservation category.